spec/support/factory/accounts/user_factory.ex

```diff
@@ -26,6 +26,11 @@ defmodule Squadster.Support.Factory.Accounts.UserFactory do
         %{user | squad_request: squad_request}
       end
 
+      def with_squad_request_to_a_certain_squad(user, squad) do
```

Comment:
Not a big deal, but maybe lets make this function name shorter like `with_request_to_squad` or `with_squad_request_to` or something like that

Reply:
The first one looks great